src/shared/Footer.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+    Container,
+    Typography,
+    Button
+} from "@mui/material";
+import React, { useRef } from "react";
+import { styled } from "@mui/styles";
+import theme from "./theme";
+
+const FooterHolder = styled('div')({
+    width: '100%',
+    height: 150,
```

Comment:
I would change the height of the footer to 50 px. The current height is too broad.